scripts/manual_operations/manual_remove.py

```diff
@@ -75,22 +75,22 @@ def multiple_versions_found(self, run_number):
         :param run_number: (int) The run number with multiple versions
         """
         # Display run_number - title - version for all matching runs
-        print("Discovered multiple reduction versions for {}{}:".format(self. instrument,
+        print("Discovered multiple reduction versions for {}{}:".format(self.instrument,
                                                                         run_number))
-        for run in self.to_delete[run_number]:
+        for run in self.to_exclude_from_deletion[run_number]:
             print("\tv{} - {}".format(run.run_version, run.run_name))
 
         # Get user input for which versions they wish to delete
-        user_input = input("Which runs would you like to delete (e.g. 1,2,3): ")
+        user_input = input("Which runs would you like to delete (e.g. 0,1,2,3 or 0-3): ")
         input_valid, user_input = self.validate_csv_input(user_input)
         while input_valid is False:
-            user_input = input('Input of \'{}\' was invalid. '
-                               'Please provide a comma separated list of values:')
+            user_input = input('Input was invalid. '
+                               'Please provide a comma separated list or a range of values: ')
             input_valid, user_input = self.validate_csv_input(user_input)
 
         # Remove runs that the user does NOT want to delete from the delete list
-        self.to_delete[run_number] = [
-            reduction_job for reduction_job in self.to_delete[run_number]
+        self.to_exclude_from_deletion[run_number] = [
+            reduction_job for reduction_job in self.to_exclude_from_deletion[run_number]
```

Comment:
I think this name is misleading now because what is left in `self.to_exclude_from_deletion` after this list comprehension is the stuff that will actually be deleted

Reply:
Good point, will change that soon